public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -34,7 +34,12 @@ export function AppChrome({ children }: Props) {
   // doesn't get re-mounted when chromeless goes from true to false.
 
   return (
-    <div className={classNames('main-view', searchBarHidden && 'main-view--search-bar-hidden')}>
+    <div
+      className={classNames('main-view', {
+        'main-view--search-bar-hidden': searchBarHidden && !state.chromeless,
+        'main-view--chrome-hidden': state.chromeless,
```

Comment:
What does chromeless mean in this context?

Reply:
@tskarhed without the header bars. so think either the login page, or any page when viewing in full kiosk mode.